app/backend/wells/models.py

```diff
@@ -1309,3 +1312,97 @@ def __str__(self):
                                                          self.screen_to)
         else:
             return 'well {} {} {}'.format(self.well, self.screen_from, self.screen_to)
+
+
+class AquiferVulnerablityCode(AuditModel):
```

Comment:
Could this class live in the aquifer project?

Reply:
There is a typo-  __Vulnerability__

Reply:
typo fixed and the 'does this not belong to aquifer rather than well' is an outstanding question I have for @LindsayMacfarlane

Reply:
The vulnerability of the aquifer is measured/assessed at the well location, so it is associated with the aquifer indirectly.  NOTE apparently these get assesses at a well location even BEFORE it is known which aquifer is being pulled from.